config.in

```diff
@@ -5,7 +5,7 @@
 # Read `man 5 sway` for a complete reference.
 
 #Session setup
-exec swayidle
+exec swayidle -i 600 -l 300 -x
```

Comment:
Take this out of the default config, or at least comment it by default. If you choose the latter extend the comment to explain what these args do and suggest the man page for further reading.

Reply:
So, there shoud not be any dpms/lockscreen handling by default?

Reply:
Correct